specification/e-referrals-service-api.yaml

```diff
@@ -537,9 +537,133 @@ paths:
           $ref: 'components/schemas/responses/Forbidden.yaml'
         '500':
           $ref: 'components/schemas/responses/InternalServerError.yaml'
+  /STU3/v1/Binary:
+    post:
+      security:
+        - bearerAuth: []
+      description: |
+        ## Overview
+        
+        Use this endpoint to upload a file containing clinical information to e-RS ready to be associated with a referral.
+
+        ## Pre-requisites
+        In order to use this endpoint you must be an authenticated e-RS user and use one of the following e-RS roles:
+          - `REFERRING_CLINICIAN`
+          - `REFERRING_CLINICIAN_ADMIN`
+          - `SERVICE_PROVIDER_CLINICIAN`
+
+        You can only upload file formats that are supported by e-RS. Currently, the supported file formats are:
+
+        | File type          | File extension(s)           |
+        | ------------------ | --------------------------- |
+        | Plain Text         | `.txt`                      |
+        | HTML Text          | `.html,` `.htm`             |
+        | PDF                | `.pdf`                      |
+        | XML Text           | `.xml`                      |
+        | RTF Text           | `.rtf`, `.rtx`              |
+        | Basic Audio        | `.pdf`                      |
+        | MPEG audio layer 3 | `.mp3`                      |
+        | PNG Image          | `.png`                      | 
+        | GIF Image          | `.gif`                      |
+        | JPEG Image         | `.jpg,` `.jpe`, `.jpeg`     |
+        | TIFF Image         | `.tif`, `.tiff`             |
+        | MPEG Video         | `.mpg`, `.mpeg`, `.mpe`     |
+        | MSWORD             | `.doc`, `.docx`             |
+
+        Currently, the maximum file size supported by e-RS is 5MB.
+
+        ## Use case
+
+        As an authenticated e-RS user
+
+        I need to upload a referral letter or other clinical information file to e-RS
+
+        So that I can associate the file with an existing referral.
+
+        ## Important notes:
+
+        This endpoint **only** performs the file upload. The uploaded file is **not** associated with a referral until the file has been associated with a referral, for example using [[HYPERLINK_A012]].
```

Comment:
This sentence seems a bit clunky

Reply:
We can't change the text in overview section at the moment, it was already refined and approved by BA's and Business. It'd have to go on another loop if we were to update it.  There is going to be a final review of full content before this gets published.